src/sbt-test/git-versioning/find-tag-newest/test

```diff
@@ -1,6 +1,8 @@
 > git init
 > git config user.email "test@jsuereth.com"
 > git config user.name "Tester"
+> git config core.hookspath /dev/null
+> git config commit.gpgsign false
```

Comment:
Some scripted tests started to take a very long time. I figured it was wainting for a GPG password prompt, so I added these lines I added before in a plugin of my own.